backend/datasources/erp.py

```diff
@@ -121,8 +121,8 @@ def erp_invoice_pdf(username: str, invoice_number: str) -> InvoicePdf:
 
 def erp_production_data(
         username: str,
-        first_timestamp_utc: AwareDatetime,
```

Comment:
Even though AwareDatetime is a datetime in python, it is indeed serialized as a string constrained to be a iso format date with the timezone informed. Using it has many benefits like ensuring timezone awarenes and transparently convert from string to datetime. Did it gave you any problem?